indexer/variables.tf

```diff
@@ -162,6 +162,12 @@ variable "full_node_container_chain_home" {
   description = "Full-node's home directory for the chain. Used to boot up the chain, and configure the `cmd` in ECS"
 }
 
+variable "full_node_root_block_device_size" {
+  type        = number
+  description = "Size of root block device in gigabytes"
+  default     = 3000
```

Comment:
what's the ideal default size? I assume we will want a smaller number? if so, can you update the default to be smaller so we don't have to override it everywhere?

Reply:
1TB should be enough.
Fixed in https://github.com/dydxprotocol/v4-infrastructure/pull/122